proposals/csharp-11.0/low-level-struct-improvements.md

```diff
@@ -1174,7 +1174,7 @@ This snippet wants to mutate a parameter by escaping elements of the data. The e
 
 ### Fun Samples
 
-#### ReadOnlySpan<T>
+#### ReadOnlySpan\<T>
 
 ```c#
 public readonly ref struct ReadOnlySpan<T>
```

Comment:
Thanks @Youssef1313. Since you're making changes here, perhaps also remove the `<T>` from the constructor signature below since the syntax is incorrect.